keyboards/mint60/serial_config.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SOFT_SERIAL_CONFIG_H
+#define SOFT_SERIAL_CONFIG_H
+
+/* Soft Serial defines */
+#define SERIAL_PIN_DDR DDRD
+#define SERIAL_PIN_PORT PORTD
+#define SERIAL_PIN_INPUT PIND
+#define SERIAL_PIN_MASK _BV(PD2)
```

Comment:
If this isn't PD0, then it can't sure serial and i2c, IIRC.  And in that case, you should only use one or the other. 

Unless I'm mis-reading.

Reply:
Thanks @mtei .
You are right.